mobile/src/screens/chat/chat.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+
+import { InputSearch, ScrollView, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { useState } from '#libs/hooks/hooks';
+
+import { ChatItem, ChatLink } from './components/components';
+import data from './data.json';
```

Comment:
move json to chat/libs/constants and use plain javascript, also start it with mocked e.g. mockedChats

Reply:
Done